python/requirements.txt

```diff
@@ -1,4 +1,6 @@
 Cython==0.29.35
+attrs==23.1.0
+cattrs==23.1.2
```

Comment:
Do we need cattrs?

Reply:
nope thanks, not in this one, it was a holdover from the next PR